### DISCUSSION TITLE
tiptap not headless!?

### DISCUSSION THREAD

**Discussion by Kukuschi** [ueberdosis/tiptap#2332] [General Questions] [answered]
https://tiptap.dev/guide/styling
"tiptap is headless, that means there is no styling provided. That also means, you are in full control of how your editor looks. The following methods allow you to apply custom styles to the editor."

That seems to not be true.
When i add css to .ProseMirror (e.g.: font-variant-ligatures: normal;)  (using Vue 3) some of it gets overwritten by 

```
<style data-tiptap-style="">

[...]

.ProseMirror {
  word-wrap: break-word;
  white-space: pre-wrap;
  white-space: break-spaces;
  -webkit-font-variant-ligatures: none;
  font-variant-ligatures: none;
  font-feature-settings: "liga" 0; /* the above doesn't seem to work in Edge */
}

[...]
```

Help how to prevent this would be much appreciated.

**Comment by hanspagel** [comment, marked as answer]
https://tiptap.dev/api/editor#inject-css

**Reply by Kukuschi** [reply]
Ah thx :)
Would be good to have a short note about that at https://tiptap.dev/guide/styling
